### PR TITLE
Update babylon.math.ts

### PR DESCRIPTION
**Pull request by dad72**
Add Vector3.One() and Vector3.Down()

### CODE REVIEW

File: src/Math/babylon.math.ts [hunk 1]
```diff
@@ -812,10 +812,18 @@
         public static Zero(): Vector3 {
             return new Vector3(0, 0, 0);
         }
+        
+        public static One(): Vector3 {
+            return new Vector3(1.0, 1.0, 1.0);
+        }
```
Comment (deltakosh) [inline src/Math/babylon.math.ts]: I'm sure about the name as One is not normalized here

**Comment by dad72** [comment]
What mean? You want another name?

**Comment by deltakosh** [comment]
Yes because for me One means normalized

**Comment by dad72** [comment]
I see no other name. for me it is a one vector3

For exemple: http://docs.unity3d.com/ScriptReference/Vector3.html

Otherwise ago "Vector3.An();" but this is less common.

**Comment by RaananW** [comment]
At what point do you need a 1,1,1 vector?

**Comment by dad72** [comment]
Hi, Raanan,
mesh.ellipsoid = new BABYLON.Vector3(1.0, 1.0, 1.0);

**Comment by dad72** [comment]
PR forget that it was not important. I no other name anyway as One()

**Pull request closed without merging**